com.playeveryware.eos/Runtime/Core/Config/PlatformConfig.cs

```diff
@@ -268,6 +268,10 @@ protected PlatformConfig(PlatformManager.Platform platform) :
 
         #region Logic for Migrating Override Values from Previous Structure
 
+        // This warning is suppressed because while EOSConfig is marked as 
+        // obsolete - it is important that it remain and be used within this
+        // section of code so that things can be properly migrated.
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
Needs a pragma restore

Reply:
Running a find all text within `PlatformConfig.cs` reveals that there is a restore directive and that it comes after the disable directive. Can you confirm?

![image](https://github.com/user-attachments/assets/186711ef-4831-45b8-a318-30944c7bf34f)

Reply:
I'm going to err on the side of moving fast and breaking things - since this isn't a PR into `development`.  Leaving conversation unresolved - which does not block merging.